Dropp/server.js

```diff
@@ -22,57 +22,105 @@ var db = admin.database(); // Get the db object to call on
 
 // Middleware
 router.use( (req, res, next) => {
-    console.log('Request came in');
+    console.log(req.method + ' request came in');
     next();
 });
 
-/* --------------- Routes --------------- */
+/* ----- Routes ----- */
 router.get('/', (req, res) => {
-    res.status(200).send('Welcome to Dropp');
+    res.send('Welcome to Dropp');
 });
 
 app.use('/api', router);
 
 // Get all dropps
 router.route('/dropps')
-  .get( (req, res) => {
+  .get((req, res) => {
     var ref = db.ref("/");
     ref.once("value", (snapshot) => {
       res.json(snapshot.val());
     });
   });
 
+// Get a specific dropp
```

Comment:
Client has the ability to request individual dropp. Dropp is suppose to be constantly refreshing, just in case of time-out scenario.